.github/workflows/unit-tests.yml

```diff
@@ -43,3 +43,25 @@ jobs:
               --env=CFLAGS='-O2 -gdwarf-4' \
               --image-tag='${{ matrix.startarg.tag }}' \
               --make '${{ matrix.startarg.make }}'
+
+  pytest:
+    runs-on: ubuntu-latest
+    permissions: {}
+    timeout-minutes: 10
+    env:
+      TOX_TESTENV_PASSENV: FORCE_COLOR
+      FORCE_COLOR: 1
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
```

Comment:
Strong veto. Please test the *real* thing (python and installed libraries) from at least one of our supported OSes. This can be the unittest container.

Such artificial environments are irrelevant for telling us whether our code actually works (e.g. on CentOS 8 or RHEL 9 or current Python in Fedora 36), and it's also not reproducible locally for development.

Reply:
OK.  I think one of our containers would be fine, in that case.  I'll look into that.  The Python setup in the `ubuntu-latest` VM on GitHub actions is positively ancient.